ansible/library/show_interface.py

```diff
@@ -194,11 +194,15 @@ def collect_interface_status(self, namespace=None, include_internal_intfs=False)
 
         return
 
-    def collect_interface_counter(self):
+    def collect_interface_counter(self, namespace=None, include_internal_intfs=False):
         regex_int = re.compile(r'\s*(\S+)\s+(\w)\s+([,\d]+)\s+(N\/A|[.0-9]+ B/s)\s+(\S+)\s+([,\d]+)\s+(\S+)\s+([,\d]+)\s+([,\d]+)\s+(N\/A|[.0-9]+ B/s)\s+(\S+)\s+([,\d]+)\s+(\S+)\s+([,\d]+)')
         self.int_counter = {}
+        cli_options = " -n {}".format(namespace) if namespace is not None else ""
+        if include_internal_intfs:
```

Comment:
can you also check that namespace exists? similar to the check in collect_interface_status to enable pytest runs on 201811 images